README.md

```diff
@@ -42,6 +42,8 @@ optional arguments:
   --no-multiline        convert only single line expressions
   -ll LINE_LENGTH, --line-length LINE_LENGTH
                         for expressions spanning multiple lines, convert only if the resulting single line will fit into the line length limit. Default value is 88 characters.
+  -n, --dry-run         Do not change the file in-place. Useful when running
```

Comment:
why -n? I am using the first letter as a short-hand for each flag; -d would be more consistent (and intuitive imo)